pkg/internal/computeresources/transformer_test.go

```diff
@@ -310,7 +310,7 @@ func TestTransformerOneContainer(t *testing.T) {
 			Containers: []corev1.Container{{
 				Resources: corev1.ResourceRequirements{
 					Limits: corev1.ResourceList{
-						corev1.ResourceMemory:           resource.MustParse("4Gi"),
+						corev1.ResourceMemory:           resource.MustParse("2Gi"),
```

Comment:
Thanks Austin! Nice job catching this bug, it looks like the behavior here wasn't consistent with our [documentation](https://github.com/tektoncd/pipeline/blob/main/docs/compute-resources.md#limits). @vdemeester could you double check me here?

Reply:
I think the init container memory limits need to be changed too